admin/data/sql/migrations/20190320145618.do.grant-census-import-user-permissions.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+const config = require('../../../config')
+
+module.exports.generateSql = function () {
+  if (config.Sql.Azure.Scale) {
+    return `GRANT CREATE TABLE ON schema::[mtc_admin] to [${config.Sql.PupilCensus.Username}];`
```

Comment:
is this enough and do they qualify for insert and update on tables by default?

Reply:
@GuyHarwood data insertion confirmed working. Update not tested since we haven't reached the stage where we update the pupil table.

Reply:
needs testing on azure.  you are defaulting to sysadmin (sa) in local so everything works regardless

Reply:
@GuyHarwood updated that migration with insert and update permissions aside from create table.